### PR TITLE
add flash messages in case user is verified or not (#235)

### PR DESCRIPTION
**Pull request by Sayok**

### CODE REVIEW

File: app/controllers/visit_requests_controller.rb [hunk 1]
```diff
@@ -32,4 +32,8 @@ def event
   def visit_request
     @visit_request ||= VisitRequest.find(params[:id])
   end
+
+  def user_verification_status
+    current_user.verified? ? :success_for_verified : :success_for_newbies
```
Comment (sveredyuk) [inline app/controllers/visit_requests_controller.rb:37]: business logic in controller detected ⚠️

Comment (denys-medynskyi) [reply app/controllers/visit_requests_controller.rb:37]: What do you suggest?

Comment (sveredyuk) [reply app/controllers/visit_requests_controller.rb:37]: I suggest using interactors. Something between controlelr features (params, flash, status and redirects) and business logic. At lead app/interactors folder already created

**Comment by denys-medynskyi** [comment]
Next time include testing instructions & link to the issue or the Trello ticket.
However it will be fixed if you will pull new development - it will have proper PR template.

File: app/controllers/visit_requests_controller.rb (hunk 1 above)
Comment (denys-medynskyi) [reply app/controllers/visit_requests_controller.rb:37]: We need to add this into the readme

Comment (sveredyuk) [reply app/controllers/visit_requests_controller.rb:37]: Ok.

Comment (denys-medynskyi) [reply app/controllers/visit_requests_controller.rb:37]: Actually I really don't get how this will improve this code.
Can you give a pseudo-code example?

Comment (sveredyuk) [reply app/controllers/visit_requests_controller.rb:37]: Have no time now to code. Let's merge and discuss this problem personally latter

**Review by denys-medynskyi — APPROVED** [review]
👍

**Pull request merged**